src/ds/ds-motion-common.cpp

```diff
@@ -0,0 +1,90 @@
+// License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Consider using a common code for ds5 / ds6

Reply:
done